### ISSUE TITLE
'DataLoader' object is not subscriptable

### ISSUE THREAD

**Issue by litaPhsinaM** [litaPhsinaM/Object-Detection-on-VR_Manish-Patil#9]

**Comment by litaPhsinaM** [comment]
DataLoader(dataset, batch_size=1, shuffle=False, sampler=None,
           batch_sampler=None, num_workers=0, collate_fn=None,
           pin_memory=False, drop_last=False, timeout=0,
           worker_init_fn=None, *, prefetch_factor=2,
           persistent_workers=False)
